application/investigate.py

```diff
@@ -0,0 +1,77 @@
+#! /usr/bin/env python3
+""" Standalone script for evaluating a dataset.
+Calculates measures of label quality and tries to spot outliers.
+
+Usage:
+    python3 investigate.py <filename>"""
+
+import argparse
+from collections import defaultdict
+import pandas as pd
+
+class Investigator(object):
+    """Utility class for calculating statistics on a data set."""
+
+    def __init__(self):
+        """Initializes basic statistics."""
+        self.entry_count = 0
+        self.artists = set([])
+        self.genres = defaultdict(int)
+        self.word_counts = defaultdict(int)
+        self.year_counts = defaultdict(int)
+        self.durations = []
+
+    def proccess_row(self, row):
```

Comment:
Typo, should only have 1 'c': not a huge deal but worth noting (also, same typo in the docstring)

Reply:
I'll fix it